admin/fabfile.py

```diff
@@ -160,6 +162,16 @@ def setup_config(ctx, app_dir, user, secret):
         )
 
 
+@task
+def wipe_db(ctx, user):
+    create_directory(ctx, "/var/backups/zam", owner=user)
+    ctx.sudo(
+        f"cp --backup=numbered /var/lib/zam/repondeur.db /var/backups/zam/repondeur.db",
```

Comment:
Is that `--backup` option standard? (I don't have it on my machine)

Reply:
Standard in GNU `ls`, but not in the BSD-flavored one you get on macOS...